lib/compiler/index.js

```diff
@@ -125,6 +125,8 @@ const compiler = {
 
     switch (options.output) {
       case "parser":
+        // This is core to this library.
+        // eslint-disable-next-line no-eval
```

Comment:
Just a suggestion. The comment [could be part](https://eslint.org/docs/latest/use/configure/rules#comment-descriptions) of `eslint` command. Probably there exists a tool that can display all such commands with their comments.